app/javascript/chat/__tests__/requestManager.test.jsx

```diff
@@ -46,7 +46,7 @@ describe('<RequestManager />', () => {
     );
 
     expect(
-      queryByText('You have no pending invitations/Joining Requests.'),
-    ).toBeDefined();
+      queryByText('You have no pending invitations.'),
```

Comment:
(non-blocking suggestion) - instead of asserting a double negative I'd suggest using `getByText` instead of `queryBy...` (since it throws an error if not found) and `toBeInTheDocument()` to verify that it's currently rendered

Reply:
@aitchiss That's a great suggestion! Just pushed a new commit with that change instead. Thanks :smile: